manifests/init.pp

```diff
@@ -201,6 +205,7 @@
 #
 class datadog_agent(
   $dd_url = 'https://app.datadoghq.com',
+  $site = 'datadoghq.com',
```

Comment:
could you define this here: `datadog_agent::params` and use the default as `$datadog_agent::params::default_site`, or something along those lines?

Reply:
Okay, I was confused because some config options are defined as we say and some as I've done. I wasn't sure which one was the better

Reply:
Basically I tend to put in params stuff that I consider would be a global const, whereas I'd just put it in `init.pp` if it's just a sane default. I agree it's a blurry line.